### PR TITLE
Tauma - Taus in Yuma

### PR DESCRIPTION
**Pull request by nikitapond**
## Summary

This pull request introduces the following changes

* Adds support for plotting taus as a background
* Adds support (requires atlas-ftag-tools update) for tautagging plots in yuma
* As a side effect of the above changes, can now also trivially plot Xbb plots. An example config is provided

## Conformity
- [ ] [Changelog entry](https://github.com/umami-hep/puma/blob/main/changelog.md)
- [ ] [Documentation](https://umami-hep.github.io/puma/)

### CODE REVIEW

**Comment by nikitapond** [comment]
Tests are failing as we require [atlas-ftag-tools !81](https://github.com/umami-hep/atlas-ftag-tools/pull/81)

File: puma/hlplots/results.py [hunk 1]
```diff
@@ -190,8 +199,8 @@ def check_nan(data: np.ndarray) -> np.ndarray:
             if tagger not in self.taggers.values():
                 self.add(tagger)
             tagger.output_flavours = self.flavours
-            if "ftau" in tagger.fxs:
-                tagger.output_flavours += [Flavours.taujets]
+            # if "ftau" in tagger.fxs:
+            #     tagger.output_flavours += [Flavours.taujets]
```
Comment (samvanstroud) [inline puma/hlplots/results.py:203]: can we remove these?

**Review by samvanstroud — COMMENTED** [review]
Looks good! I made a new release for the tools package and had a small comment

**Review by samvanstroud — APPROVED** [review]
Thanks Nikita!

**Pull request merged**